test/abstractarray.jl

```diff
@@ -1280,3 +1280,22 @@ end
     @test Int[0 t...; t... 0] == [0 1 2; 1 2 0]
     @test_throws ArgumentError Int[t...; 3 4 5]
 end
+
+@testset "issue #39896, modified getindex " begin
+    #Testing types holds for the range types in Base that use 1-based indexing
+    for arr = ([1:10;], reshape([1.0:16.0;],4,4), reshape(['a':'h';],2,2,2))
+        for inds = (2:5, Base.OneTo(5), BigInt(3):BigInt(5), UInt(4):UInt(3))
+            @test arr[inds] == arr[collect(inds)]
+            @test eltype(arr[inds]) === eltype(arr)
+            @test ndims(arr[inds])==1
+            @test arr[inds] isa AbstractVector{eltype(arr)}
```

Comment:
You don’t need all three of these, either the first two or the last one is enough.

Reply:
Changes done.